### ISSUE TITLE
[use-strict] and external modules

### ISSUE THREAD

**Issue by Bartvds** [palantir/tslint#108]
I'm converting some internal modules (with `'use strict';` inside the main module) to simple node.js-style external modules. 

The source for the external modules should have 'use strict', and first-thing ~~*after*~~ before any `import`'s ~~so it ends up on top inside the AMD call:~~

````js
'use strict';

import fs = require('fs');

function foo() {
    // ...
}
export = foo;
````

Current rule doesn't support this and flags any function or sub-module.

**Comment by adidahiya** [comment]
`use-strict` will soon be obsolete, see #678
